### ISSUE TITLE
Rename variadic parameter is not break BC

### ISSUE THREAD

**Issue by vjik** [Roave/BackwardCompatibilityCheck#733]
```php
interface A {
    public function test(...$a): void;
}
```
change to
```php
interface A {
    public function test(...$b): void;
}
```

This is not break BC.

All data pass to variadic parameter when named arguments used. For example: https://3v4l.org/stRQl

**Comment by Ocramius** [comment]
Could we have that in a test case, perhaps? 🤔

**Comment by vjik** [comment]
Example of false-positive result: https://github.com/yiisoft/data/actions/runs/3999454619/jobs/6863373008

**Comment by Ocramius** [comment]
I would need a patch touching these files:

https://github.com/Roave/BackwardCompatibilityCheck/blob/47df2c2862cb1110260843412745827ce5a28b14/src/DetectChanges/BCBreak/FunctionBased/ParameterNameChanged.php#L36-L75
https://github.com/Roave/BackwardCompatibilityCheck/blob/47df2c2862cb1110260843412745827ce5a28b14/test/unit/DetectChanges/BCBreak/FunctionBased/ParameterNameChangedTest.php#L22-L23
